### ISSUE TITLE
Somfy SupportedManufacturerSettingsCommands messed up in more-info

### ISSUE THREAD

**Issue by Mariusthvdb** [home-assistant/frontend#9838]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

SupportedManufacturerSettingsCommands in More-info are not listed correctly (as in the dev states)

### Describe the behavior you expected

correct listing

### Steps to reproduce the issue

1. check any cover and click for more-info
2.
3.
...

### What version of Home Assistant Core has the issue?

2021.8.x

### In which browser are you experiencing the issue with?

Any really really, both desktop and handheld and iOS app

### Additional information

More info:

<img width="400" alt="Schermafbeelding 2021-08-18 om 11 29 17" src="https://user-images.githubusercontent.com/33354141/129875331-dd3f7698-d665-4538-b587-829910d86515.png">

dev tools state:

<img width="604" alt="Schermafbeelding 2021-08-18 om 11 31 05" src="https://user-images.githubusercontent.com/33354141/129875400-a64ed224-3b1c-4439-bfe9-d6f1fe3c8f84.png">

**Comment by ludeeus** [comment]
This is an integration issue, none of these attributes should even be present on the entity.

**Comment by Mariusthvdb** [comment]
ok thanks, did already bring the author up to speed. Still, if the dev tools/state shows alright, like it does, how can it still be an issue for the more-info?
asking so I can give more feedback

**Comment by ludeeus** [comment]
space, there is just not enough room to show the data due to long names

**Comment by iMicknl** [comment]
I already wrote it on the community forum, but let's do it here as well for transparency.

This is part of the ha-tahoma integration. These states / attributes will be removed soon, currently they are still present in the custom component to help users spot available states which could be transformed to a sensor or binary sensor. We already migrated from using state for RSSI and battery to separate sensors, as advised for new core integrations.
